pipeline.yaml

```diff
@@ -23,10 +23,11 @@ data:
   # Datasets with other formats, including `csv`, can be used by implementing and
   # specifying a `custom_loader_method`
   custom_loader_method: steps.ingest.load_file_as_dataframe
-  # If the `spark_sql` `format` is specified, the `sql` entry is used to specify a SparkSQL
-  # statement that identifies the dataset to use
-  sql: SELECT * FROM delta.`{{INGEST_DATA_LOCATION}}`
-  # If the `delta` `format` is specified, you can also configure the Delta table `version` to read
+  # If the `spark_sql` `format` is specified,
+  # 1. with dbfs location format `sql` entry is used to specify a SparkSQL statement that identifies the dataset to use
+  # sql: SELECT col1, col2 FROM delta.`{{INGEST_DATA_LOCATION}}`
+  # 2. with table location format `sql` entry is used to specify a SparkSQL statement that identifies the dataset to use
```

Comment:
ditto here:
`If the table location format is table-like, use the following sql command for Spark to read sub-columns from the table:`
` # sql: ...`